reader.go

```diff
@@ -1346,7 +1346,7 @@ func (r *reader) run(ctx context.Context, offset int64) {
 
 			case errors.Is(err, UnknownTopicOrPartition):
 				r.withErrorLogger(func(log Logger) {
-					log.Printf("failed to read from current broker for partition %d of %s at offset %d, topic or parition not found on this broker, %v", r.partition, r.topic, toHumanOffset(offset), r.brokers)
+					log.Printf("failed to read from current broker for partition %d of %s at offset %d, topic or parition not found on this broker, %v: %v", r.partition, r.topic, toHumanOffset(offset), r.brokers, err)
```

Comment:
It looks like with these new formats we'll have 
`failed to read from current broker for partition %d of %s at offset %d, topic or parition not found on this broker, %v: Unknown Topic Or Partition`

Could we possible reformat these a bit so we don't log redundant messages?

Potentially like
`
failed to read from current broker for partition %d of %s at offset %d, Unknown Topic Or Partition, %v`